shell/platform/android/io/flutter/embedding/engine/renderer/FlutterRenderer.java

```diff
@@ -95,6 +95,12 @@ public void removeIsDisplayingFlutterUiListener(@NonNull FlutterUiDisplayListene
   public SurfaceTextureEntry createSurfaceTexture() {
     Log.v(TAG, "Creating a SurfaceTexture.");
     final SurfaceTexture surfaceTexture = new SurfaceTexture(0);
+    return registerSurfaceTexture(surfaceTexture);
+  }
+
+  /** Registers and returns a {@link SurfaceTexture} that is also made available to Flutter code. */
```

Comment:
can this also talk about the ownership of `SurfaceTexture`? For example, what happens if a user of this API calls, `release()`, and then embedding also calls `release()` internally.  From the docs:

"release() frees all the buffers and puts the SurfaceTexture into the 'abandoned' state. Once put in this state the SurfaceTexture can never leave it. When in the 'abandoned' state, all methods of the IGraphicBufferProducer interface will fail with the NO_INIT error."

Reply:
Done